omod/src/main/webapp/fragments/dashboardwidgets/discreteValues.gsp

```diff
@@ -0,0 +1,23 @@
+
+<div class="info-section">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label) }</h3>
+        <a href="${ ui.urlBind("/" + contextPath + app.url, [ "patient.uuid": patient.id ]) }" class="right">
+            <i class="icon-share-alt edit-action" title="Edit"></i>
```

Comment:
Is this an "Edit"?  If not, it shouldn't be listed that way.  And...what is this exactly?  What would one actually be linking to or editing in such a widget that spans multiple obs across encounters within a program?  Is this in the requirements or is this just copied from some other fragment and you forgot to remove it?

Reply:
deleted